domains/bteamapp.json

```
@@ -3,7 +3,9 @@
     "username": "bteamapp",
     "email": "helpapp.bta@gmail.com"
   },
+  "proxied": true,
   "records": {
-    "CNAME": "ghs.google.com"
+    "CNAME": "ghs.google.com",
+    "TXT": [ "brevo-code:4a6e073acf19d6eb62ae4ef7d9259923" ]
```

its just a ingle TXT record and for readability
```suggestion
    "TXT": "brevo-code:4a6e073acf19d6eb62ae4ef7d9259923"
```